modules/oneVideoBidAdapter.js

```diff
@@ -3,7 +3,7 @@ import {registerBidder} from '../src/adapters/bidderFactory.js';
 const BIDDER_CODE = 'oneVideo';
 export const spec = {
   code: 'oneVideo',
-  VERSION: '3.0.0',
+  VERSION: '3.0.1',
```

Comment:
Your tests are failing because in tests it is expected that `VERSION` is `3.0.0`.

Did you consider changing hardcoded version to: `'$prebid.version$'`?

In that way, each new pbjs release will automatically increment version for you, you won't have to manually increment it.